openapi/ap/Platform API.yml

```diff
@@ -0,0 +1,281 @@
+        - in: query
+          name: order
+          description: Either ascending or descending by `order_by` value.
+          required: false
+          schema:
+            type: string
+            enum: [ "asc", "desc" ]
+            default: "desc"
+        - in: query
+          name: order_by
+          description: The data to be used for ordering transactions.
+          schema:
+            type: string
+            enum: [ "started_at", "completed_at", "transfer_received_at" ]
+            default: "started_at"
+        - in: query
+          name: cursor
+          description:
+            The cursor used for pagination. Each response will include a `cursor` value that can be used to
+            view the next page, assuming all other parameters kept the same.
+          required: false
+          schema:
+            type: string
+        - in: query
+          name: after
+          description: Timestamp for which all `order_by` values will be after (exclusive), in UTC.
+          required: false
+          schema:
+            type: string
+            format: date-time
+        - in: query
+          name: before
+          description: Timestamp for which all `order_by` values will be before (exclusive), in UTC.
+          required: false
+          schema:
+            type: string
+            format: date-time
+        - in: query
+          name: status
+          description: Status of the transaction.
+          required: false
+          schema:
+            type: string
+            enum: [
+              "pending_sender",
+              "pending_stellar",
+              "pending_customer_info_update",
+              "pending_receiver",
+              "pending_external",
+              "completed",
+              "expired",
+              "error"
+            ]
+      responses:
+        '200':
+          description: valid request
+          content:
+            application/json:
+              schema:
+                oneOf:
+                  - $ref: './schemas.yml#/components/schemas/AllTransactionsSEP24'
+                  - $ref: './schemas.yml#/components/schemas/AllTransactionSEP31'
+        '400':
+          description: invalid request
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  error:
+                    type: string
+                required:
+                  - error
+    patch:
+      tags:
+        - "Transactions"
+      summary: Accepts updated information on transactions.
+      operationId: patchTransactions
+      description:
+        Accepts one or more objects containing updated information on transactions. Note that requests containing
+        invalid data for any transaction will result in a 400 Bad Request and no transactions will be updated. 
+        Importantly, this operation is NOT atomic. If one of updates fail, all previous updates would be applied, 
```

Comment:
Change to: "If one update fails, all previous updates would be applied..." Lose the "of"